lib/Frontend/CompilerInvocation.cpp

```diff
@@ -460,6 +460,12 @@ static void parseAnalyzerConfigs(AnalyzerOptions &AnOpts,
     Diags->Report(diag::err_analyzer_config_invalid_input) << "ctu-dir"
                                                            << "a filename";
 
+  if (AnOpts.CTUOnDemandParsing &&
+      !llvm::sys::fs::exists(AnOpts.CTUOnDemandParsingDatabase))
+    Diags->Report(diag::err_analyzer_config_invalid_input)
+        << "ctu-on-demand-parsing-database"
+        << "a filename";
```

Comment:
Should we `<<` `AnOpts.CTUOnDemandParsingDatabase` rather?

Reply:
I think this usage is intended, as we report that the option itself contains an invalid value. (However I see the point of doing it the other way around, but then other diagnostic messages should also be updated in order to remain consistent.)

Reply:
Ok.